camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/partitioner/VintedPartitioner.java

```diff
@@ -23,11 +23,17 @@ public String encodePartition(JobContext context, IEtlKey key) {
         long outfilePartitionMs = EtlMultiOutputFormat.getEtlOutputFileTimePartitionMins(context) * 60000L;
         String portal = key.getPartitionMap().get(PORTAL).toString();
         // get output date formatter for specific portal
-        String portalTimeZone = context.getConfiguration().get(ETL_PORTAL_TIMEZONE + "." + portal, null);
+        String portalTimeZone = context.getConfiguration().get(ETL_PORTAL_TIMEZONE + "." + portal);
+
         if (portalTimeZone == null) {
-            portalTimeZone = context.getConfiguration().get(EtlMultiOutputFormat.ETL_DEFAULT_TIMEZONE, "America/Los_Angeles");
+            throw new RuntimeException("Missing timezone configuration for portal=" + portal);
```

Comment:
Well.. Passage will fail before this comes to Camus. But if it somehow comes to Camus:

````
[LocalJobRunner] - job_local662617928_0001
java.lang.Exception: java.lang.RuntimeException: Missing timezone configuration for portal=wat
        at org.apache.hadoop.mapred.LocalJobRunner$Job.runTasks(LocalJobRunner.java:462)
        at org.apache.hadoop.mapred.LocalJobRunner$Job.run(LocalJobRunner.java:522)
Caused by: java.lang.RuntimeException: Missing timezone configuration for portal=wat
        at com.linkedin.camus.etl.kafka.partitioner.VintedPartitioner.encodePartition(VintedPartitioner.java:29)
        at com.linkedin.camus.etl.kafka.mapred.EtlMultiOutputFormat.getWorkingFileName(EtlMultiOutputFormat.java:193)
        at com.linkedin.camus.etl.kafka.mapred.EtlMultiOutputCommitter.addCounts(EtlMultiOutputCommitter.java:40)
        at com.linkedin.camus.etl.kafka.mapred.EtlMultiOutputRecordWriter.write(EtlMultiOutputRecordWriter.java:100)
        at com.linkedin.camus.etl.kafka.mapred.EtlMultiOutputRecordWriter.write(EtlMultiOutputRecordWriter.java:23)
        at org.apache.hadoop.mapred.MapTask$NewDirectOutputCollector.write(MapTask.java:658)
        at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
        at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
        at com.linkedin.camus.etl.kafka.mapred.EtlMapper.map(EtlMapper.java:25)
        at com.linkedin.camus.etl.kafka.mapred.EtlMapper.map(EtlMapper.java:19)
        at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
        at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:787)
        at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
        at org.apache.hadoop.mapred.LocalJobRunner$Job$MapTaskRunnable.run(LocalJobRunner.java:243)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
        at java.util.concurrent.FutureTask.run(FutureTask.java:262)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
        at java.lang.Thread.run(Thread.java:745)
...
[JvmMetrics] - Cannot initialize JVM Metrics with processName=JobTracker, sessionId= - already initialized
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 0
        at com.linkedin.camus.etl.kafka.CamusJob.run(CamusJob.java:392)
        at com.linkedin.camus.etl.kafka.CamusJob.run(CamusJob.java:694)
        at org.apache.hadoop.util.ToolRunner.run(ToolRunner.java:70)
        at org.apache.hadoop.util.ToolRunner.run(ToolRunner.java:84)
        at com.linkedin.camus.etl.kafka.CamusJob.main(CamusJob.java:658)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:606)
        at org.apache.hadoop.util.RunJar.run(RunJar.java:221)
        at org.apache.hadoop.util.RunJar.main(RunJar.java:136)
```

Exit status - `1`.